_posts/2021-06-25-app.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title: "Unsere CargoRocket mobile App"
+author: David Prenninger
+excerpt: "Entwicklung einer multi-plattform Navigations-Applikation"
+description: "Entwicklung einer multi-plattform Navigations-Applikation"
+---
+
+Dieser Artikel soll Einblicke geben in die Idee und Entwicklung der ersten mobilen Lastenrad-Routing Applikation. Um mehr über den zugrundeliegenden Algorithmus zu erfahren lohnt sich ein Blick in unseren Artikel ["Der technische Aufbau unseres Lastenrad Routing"](https://cargorocket.de/2021/05/31/routing.html)
+Zum Ende des Artikels findet sich eine technische Beschreibung des Quellcodes der App. 
+
+## Motivation
+
+### Mobidata Hack
+Schon während des MobiData BW 2020 Hackathons, bei dem die Idee zum CargoRocket Lastenradrouting entstand, überlegten wir, wie wir das fertige Routing am besten präsentieren, validieren und am Ende auch verwendbar machen könnten. Schnell war klar, dass sich eine App für mobile Endgeräte sich besonders anbietet. Leider kam es während dem Hackathon zu [Technischen Schwierigkeiten](###Startschwierigkeiten), die uns daran hinderten schon zum Ende der 48h eine funktionsfähige App zu präsentieren. Als Ergebnis reichten wir als Routing-Demonstator eine Web-Applikation ein, die das Basisrouting auf eine einfache Weise visualisierte ([Github Repo [Archived]](https://github.com/CargoRocket/Launchpad)).
+
+<figure>
+  <img alt="Screenshot of Launchpad Web-Applikation" src="/assets/images/launchpad-1.0.png" style="max-width: 100%;" />
+  <figcaption>Screenshot der Launchpad Web-Applikation</figcaption>
+</figure> 
+
+### Der nächste Schritt
+Für den folgenden Förderzeitraum steckten wir uns ambitionierte Ziele. Innerhalb von 3 Monaten sollte eine mobile Applikation für Android und iOS entwickelt werden.
+
+Aktuell (20.06.21) befindet sich die Android version im BETA-Test.
+
+Während dieser Testphase erhoffen wir uns Feedback zur App, aber auch zu unserem Routing und unterliegenden Index.
+Direkt Tester:Innen ein Produkt zum Ausprobieren zur Verfügung zu stellen ist aus unserer Sicht eine riesige Chance hier schnell und effizient breite Rückmeldungen einholen zu können.
+Tester:Innen haben außerdem die Möglichkeit sich direkt auf der geplanten route navigieren zu lassen und vorort Hindernisse zu melden.
+
+Die iOS version der App wird im Anschluss des BETA-Tests unter Android in einigen Wochen im App-Store zum Download zur Verfügung stehen.
+
+### Datenerhebung
+
+Zur späteren Analyse zeichnet die App die Routen der Tester auf und wird diese später anonymisiert an uns übertragen. Dabei ist uns besonders wichtig, dass aus diesem Datensatz der tatsächliche Wohnort / Arbeitsstädte unserer Tester:Innen nicht hervorgeht. Hierzu wird vor Beginn der Datenübertagung hier dazu ein Artikel erscheinen. Aus diesem Datensatz lässt sich die tatsächliche Reisezeit und umfahrene Hindernisse entnehmen.
```

Comment:
ich glaube ich würde den Satz hier streichen. Wie genau wir das machen werden ist ja noch nicht klar. Wenns um Erhebung von personenbezogenen Daten geht sollte man sehr präzise sein, sonst entstehen nur weitere Fragen oder falsche Schlussfolgerungen ^^

Reply:
Soll ich es eher als Intention formuliern?

Reply:
Bitte Discussion wieder aufmachen, falls das immernoch zu konkret ist.

Reply:
was ganz anderes - weil ich das hier schon ein paar mal gesehen habe ;) : beim Gendern gibts entweder das Binnen-I (TesterInnen) oder den Doppelpunkt (Tester:innen). Nicht beides kombiniert ^^.

Ich finds immer noch schwierig. Ggf. zeichen wir zB gar nicht die ganze Route auf, sondern nur dort, wo rerouting stattfindet. "Anonymisiert übertragen" wird im zweifel auch nicht passieren, sondern die Anonymisierung wird ggf. erst bei uns zentral stattfinden (abgesehen davon, dass Anonymisierung für einzelne Datenpunkte kaum umzusetzen ist). Es gibt noch einige andere Orte die kritisch sind, außer Wohnort und Abeitsstätte (zB Kindergarten der Kinder, fitnessstudio oder das Lieblingscasino). Außerdem impliziert das, dass wir mit Data Mining verfahren versuchen Wohn- und Arbeitsorte zu ermitteln. Auch das wir daraus Hindernisse ableiten können, ist noch nicht so ganz klar (und sollte etwas weicher formuliert sein).
sorry, vielleicht bin ich hier ein bisschen zu kleinlich, vielleicht stecke ich da zu sehr in dem Thema drin ^^

Wie wärs mit: 
Um unser Lastenrad-Routing weiter zu verbessern, sollen zukünftig neben User-Feedback auch Informationen zum realen Fahrverhalten zurück in unsere Bewertung einfließen. So möchten wir über Daten der App tatsächliche Geschwindigkeiten ermitteln, oder Streckenabschnitte identifizieren, bei denen die Fahrer:innen sich für andere Wege als die vorgeschlagenen entschieden haben. Bei der Verwendung solch personenbezogener Mobilitätsdaten steht die Privatsphäre der Nutzer:innen für uns an vorderster Stelle. Vor Beginn der Datenerhebung wird hierzu ein weiterer Artikel über die genaue Anonymisierung und Verwendung der Daten erscheinen.

Reply:
Nice der Text klingt gut!
Zwecks Gendern.. existiert die Mischform nicht? Finde die verbindet ja zum einen vorlesbarkeit und eindeutiges Schriftbild.

Reply:
Wenn euch das eher stört können wir das aber sicher in : ändern.